examples/inputs_floating/gch_fixed.yaml

```diff
@@ -0,0 +1,226 @@
+
```

Comment:
The gch.yaml input file also serves as the reference for the documentation. It might be worthwhile to remove all of the comments here for readability.
Same comment for the group of input files added here.